BITSManager/SetJobPropertyControl.xaml

```diff
@@ -49,5 +49,19 @@ Licensed under the MIT License.
         <TextBox x:Name="_uiUserName" HorizontalAlignment="Stretch" AutomationProperties.LabeledBy="{Binding ElementName=_uiJobUserNameHeader}" />
         <TextBlock Text="{x:Static p:Resources.JobPassword}" x:Name="_uiJobPasswordHeader" />
         <TextBox x:Name="_uiPassword" HorizontalAlignment="Stretch" AutomationProperties.LabeledBy="{Binding ElementName=_uiJobPasswordHeader}" />
+        <Grid>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="*" />
+                <ColumnDefinition Width="auto" />
+                <ColumnDefinition Width="auto" />
+            </Grid.ColumnDefinitions>
+            <TextBlock Text="{x:Static p:Resources.JobCustomHeadersTitle}" x:Name="_uiJobCustomHeadersHeader" />
+            <Button Grid.Column="1" Content="{x:Static p:Resources.JobAddHeader}" ToolTip="{x:Static p:Resources.JobAddHeaderToolTip}" Click="OnAddNewCustomHeader"  />
```

Comment:
>ToolTip [](start = 82, length = 7)

Is there a reason for the tooltip? It doesn't exist on the other button and I Don't see tooltips used anywhere else in this file. Seems kinda random to just start adding tooltips all of a sudden.

Reply:
The other button ought to have a tooltip, too. all buttons other than standard ones like "OK" and "Cancel" and "open" should have them (it helps with accessibility)

---
In reply to: [248038002](https://github.com/Microsoft/BITS-Manager/pull/11#discussion_r248038002) [](ancestors = 248038002)